src/Command/InfectionCommand.php

```diff
@@ -458,7 +458,7 @@ private function assertCodeCoverageExists(Process $initialTestsProcess, string $
 
         $coverageDir = $config->getCoveragePath();
 
-        $coverageIndexFilePath = $coverageDir . '/' . XMLLineCodeCoverage::COVERAGE_INDEX_FILE_NAME;
+        $coverageIndexFilePath = $coverageDir . '/' . LineCodeCoverage::COVERAGE_INDEX_FILE_NAME;
```

Comment:
Isn't this file PHPUnit specific?

Reply:
No, `index.xml` is for all adapters - this is basically index file of XML report. All 3 adapters generate XML coverage reports in their own folders, but all reports has `index.xml`